AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -92,6 +92,7 @@ techadvisor.co.uk,forum.notebookreview.com,computerworld.com,skysports.com#$#bod
 !
 !
 balcaorural.com.br##.j-cookieAlert
+dlford.io###gatsby-focus-wrapper > div > footer:contains(Privacy Notice)
```

Comment:
please change to:
dlford.io#?##gatsby-focus-wrapper > div[class^="layout__Outer-sc-"] > footer:has(> section > p > div:contains(Privacy Notice))